### PR TITLE
Added conda init command in Getting Set Up

### LINKED ISSUE #11282: Document conda init in Getting Set Up
May be necessary before `conda activate`

### PR DESCRIPTION
**Pull request by afzal442**
Added init conda command into this dev_guide

All pull requests must have an associated issue in the issue tracker. If there
isn't one, please go open an issue describing the defect, deficiency or desired
feature. You can read more about our issue and PR processes in the
[wiki](https://github.com/bokeh/bokeh/wiki/BEP-1:-Issues-and-PRs-management).

- [x] issues: fixes #11282
- [x] tests added / passed
- [x] release document entry (if new feature or API change)

### CODE REVIEW

**Comment by afzal442** [comment]
Hi @bryevdv, sorry for the late acknowledgement I made here. Plz, review this PR. Thanks.

**Comment by bryevdv** [comment]
@afzal442 this needs a little work. Running `conda init` only needs to happen *once* after the Anaconda install, but before creating the envs. It does not typically ever need to be run again on the same system.

**Comment by afzal442** [comment]
Thanks @bryevdv, this makes sense now. Will update it soon.

**Comment by afzal442** [comment]
@bryevdv plz have a look at it. Thanks.

File: sphinx/source/docs/dev_guide/setup.rst [hunk 1]
```diff
@@ -84,11 +84,16 @@ At the top level of the repository, issue the following command in a terminal:
...
+Then, activate the environment:
 
 .. code-block:: sh
 
     conda activate bkdev
+    
+.. note::
+    
+    Before creating a conda environment, initialize Anaconda with this command
+    ``conda init`` only once.
```
Comment (bryevdv) [inline sphinx/source/docs/dev_guide/setup.rst:96]:
```suggestion
    Before creating a conda environment for the first time, you may need to 
    initialize Anaconda with the ``conda init`` (only once).
```

Comment (bryevdv) [reply sphinx/source/docs/dev_guide/setup.rst:96]: @afzal442 Let me know what you think off this suggestion. I wanted to make it more clear that this was a "before the first env" not "before any env"

Comment (afzal442) [reply sphinx/source/docs/dev_guide/setup.rst:96]: Ah, I see. Let me commit your suggestion. Thanks.

**Review by bryevdv — APPROVED** [review]

**Comment by bryevdv** [comment]
Thanks @afzal442 will merge when green

**Pull request merged**